Makefile

```diff
@@ -17,6 +17,9 @@ envsetup:
 	make db/setup
 	make install/package
 
+envrevoke:
```

Comment:
`revoke` would mean to remove permissions. Maybe something like `env/up` and `env/down` would make more sense 💭 Or just `envdown`